platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/cache/VirtualNodeCache.java

```diff
@@ -1434,11 +1434,11 @@ private Mutation<K, VirtualLeafRecord<K, V>> mutate(
      * @param <V>
      * 		The value type referenced by the mutation list
      */
-    private static <K, V> void purge(
+    private static <K, V> StandardFuture<Void> purge(
```

Comment:
You change the return value type but never use it.

Reply:
This is a leftover from my first approach to this issue. I was going to use this StandardFuture and call `getAndRethrow()` on it in `release()`, but gave up with this idea. `array.parallelTraverse()` returns a StandardFuture anyway